src/component/RecipeList.js

```diff
@@ -0,0 +1,39 @@
+import { useState } from "react";
+import RecipeListItem from "./RecipeListItem";
+import Pagination from "./Pagination";
+
+function RecipeList({ recipes, handleChooseRecipe }) {
+  const [currentPage, setCurrentPage] = useState(1);
+  //   const [recipeList, setRecipeList] = useState([
+  //     ...recipes.slice(startIndex, endIndex),
+  //   ]);
+  const [recordsPerPage] = useState(10);
```

Comment:
Why is `useState` here? If it never changes, declare it as a constant instead.

Reply:
https://www.perssondennis.com/articles/react-anti-patterns-and-best-practices-dos-and-donts#user-content-use-usestate-instead-of-variables